tests/sentry/grouping/test_normalize_message.py

```diff
@@ -122,12 +122,12 @@ def record_analytics(monkeypatch):
         (
             "Uniq ID - sql savepoint",
             '''SQL: RELEASE SAVEPOINT "s140177518376768_x2"''',
-            '''SQL: RELEASE SAVEPOINT "<uniq_id>"''',
+            """SQL: RELEASE SAVEPOINT <uniq_id>""",
         ),
         (
             "Uniq ID - api gateway",
             """API gateway VdLchF7iDo8sVkg= blah""",
-            """API gateway <uniq_id>= blah""",
+            """API gateway <uniq_id> blah""",
```

Comment:
I think we should uncomment these lines so we don't forget at a later stage. What do you think?
https://github.com/getsentry/sentry/blob/c153cc529b29030157b9f0d609205335778965ed/tests/sentry/grouping/test_normalize_message.py#L142-L151

Reply:
The uniqID ones all pass now, so I uncommented them. I moved the rest to be `pytest.mark.xfail` to clarify that we should fix them eventually.